templates/techreport/components/timeseries.html

```diff
@@ -39,13 +39,15 @@ <h4>
               <div class="breakdown-item">
                 <p class="breakdown-label">{{ breakdown.name }}</p>
                 <p class="breakdown-value">00{% if breakdown.suffix == "%" %}.00{% endif %}{{ breakdown.suffix }}</p>
+                <p class="breakdown-value-secondary"></p>
```

Comment:
Is this only needed because the chart is always in bytes? TBH I don't love the redundancy and I think we can get away with only showing the formatted value. Alternatively, WDYT about stuffing the # of bytes into the `title` attribute of the formatted value?

<img width="482" alt="image" src="https://github.com/HTTPArchive/httparchive.org/assets/1120896/371aeac9-0c0e-4367-bb6a-87fe7697bdc1">

Reply:
I removed it for now since the `title` comes with accessibility challenges and is easily overlooked anyway. If people need to hover to see the value they can just as well hover over the graph (which works with keyboard etc) 😄

If we get feedback that it's confusing that one is in MB/KB and the other one in bytes, then maybe we can add a custom (accessible) tooltip on the formatted values.